src/OasisCdpManager.sol

```diff
@@ -57,17 +47,16 @@ contract OasisCdpManager is DSNote {
 
         address urn = address(new UrnHandler(vat));
         urns[usr][ilk] = urn;
-        emit NewCdp(msg.sender, usr, ilk, urn);
+        emit NewCdp(usr, ilk, urn);
```

Comment:
We are emitting a NewCdp() here, but this function also has a `note` modifier.  We should probably not have duplicate events.  We may want to remove the `note`?

Reply:
to be decided later.